Doppler.Jobs.Server/Startup.cs

```diff
@@ -97,6 +99,12 @@ private static IAsyncPolicy<HttpResponseMessage> GetRetryPolicy(int retry)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Information()
+                .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+                .WriteTo.File("Logs\\DopplerJobs.log", rollingInterval: RollingInterval.Day)
+                .CreateLogger();
```

Comment:
Is it possible to also continue writing to _stdout_?

Reply:
Yes done in this commit https://github.com/MakingSense/doppler-jobs/pull/3/commits/0734fe3ceedb6a0392995b4357de89dbb0d3850f

![screen](https://user-images.githubusercontent.com/6796523/78294193-4e294a00-7500-11ea-8ea4-f32f1e71f80d.PNG)